vscode/microsoft-kiota/src/extension.ts

```diff
@@ -61,43 +68,14 @@ export async function activate(
             return;
           }
         }
-        if (uri.path.toLowerCase() === "/openmanifest") {
```

Comment:
this is going to break the integration with Graph Explorer. I don't have any issue with it. But I want to make sure that @sebastienlevert is aware of this.

Reply:
Fully aware.

Reply:
I might be missing some context here. Can I assume the integration with GE will be removed and that it's safe to remove support for it here?

Reply:
The feature was never shipped. You can go and see that it's not available on the current version of GE (https://developer.microsoft.com/en-us/graph/graph-explorer)

Reply:
![SurprisedPikachuGIF](https://github.com/microsoft/kiota/assets/7905502/af0c34cd-90ea-49f0-b569-ec2aed622abe)